packages/cli/src/lifecycles/serve.js

```diff
@@ -69,9 +69,8 @@ async function getDevServer(compilation) {
       for (const plugin of resourcePlugins) {
         if (plugin.shouldServe && (await plugin.shouldServe(url, request))) {
           const current = await plugin.serve(url, request);
-          const merged = mergeResponse(response.clone(), current.clone());
 
-          response = merged.clone();
+          response = mergeResponse(response.clone(), current.clone());
```

Comment:
I notice that some of these use `current.clone()` and some don't. Is that intentional, or should they all be the same way? I've left it as is for now, but I can make them all the same while I'm here.